setup.py

```diff
@@ -1,17 +1,88 @@
-import os, sys
 from setuptools import setup, find_packages
+import os
+import shutil
+import sys
 
-setup_args = {}
+# TODO: build time deps to declare when we start packaging
+import pyct.build
+import param.version
+
+########## dependencies ##########
+
+# NOTE: This is just a start; you should ignore these for now. We are
+# currently using dependencies.txt, which can only be installed using
+# conda. The split between required and extras is also currently
+# arbitrary.
```

Comment:
I'd vote for removing all of this incomplete code; we can recreate it when we're ready to support installation via pip.